lib/plugin/autoDelay.js

```diff
@@ -3,7 +3,7 @@ const store = require('../store');
 const recorder = require('../recorder');
 const event = require('../event');
 const log = require('../output').log;
-const supportedHelpers = require('./standardActingHelpers');
+const supportedHelpers = require('./standardActingHelpers').slice();
```

Comment:
what about modern spread operator usage? `supportedHelpers = [...require('./standardActingHelpers')]?

Reply:
Hi @elukoyanov, 

Because I don't know much about Node Js, I tried to fix the problem as simply as possible.

Reply:
Well, this is fine as well

Reply:
Previously this `slice` was there, not sure why it was removed